src/main/webapp/app/entities/programming-exercise/programming-exercise.component.html

```diff
@@ -8,7 +8,7 @@ <h4>
             Link new Programming Exercise
             </span>
         </button>
-        <button *ngIf="course" id="jh-create-entity" class="btn btn-primary float-right jh-create-entity create-programming-exercise mr-1" [routerLink]="['new']">
```

Comment:
why did you change this?

Reply:
This change is needed also for the quiz in other case the routing does not work with the create button